Use dockerize for `make db` in integration tests

If the DB doesn't start up right away, you'll get a failure:

```
Creating integration-tests_db_1 ... done
docker-compose exec db ./reset.sh
psql: could not connect to server: No such file or directory
	Is the server running locally and accepting
	connections on Unix domain socket "/var/run/postgresql/.s.PGSQL.5432"?
make: *** [db] Error 2
```

Using `jwilder/dockerize` would allow us to hold until the DB's up and ready.

--- a/packages/integration-tests/Makefile
+++ b/packages/integration-tests/Makefile
@@ -11,7 +11,7 @@ build:
 
 # Create/recreate the database for applying migrations from scratch.
 db:
-	docker-compose up -d db
+	docker-compose up db-wait
 	docker-compose exec db ./reset.sh
 	npm run migrate
 	npm run codegen

